decidim-core/app/helpers/decidim/layout_helper.rb

```diff
@@ -33,7 +33,10 @@ def icon(name, options = {})
       html_properties["aria-label"] = options[:aria_label]
       html_properties["role"] = options[:role]
       html_properties["aria-hidden"] = options[:aria_hidden]
-      html_properties["class"] = "icon icon--#{name} #{options[:class]}"
+
+      icon_class = "icon"
+      icon_class = "" if options[:removeIconClass]
```

Comment:
@Dor3nz you can always PR `decidim-design`, is what I've done in the past to unify styles. Looks like yak shaving but better keep everything in shape to reduce friction.